app/services/plans/update_service.rb

```diff
@@ -74,6 +82,18 @@ def bill_charges_monthly?
       params[:bill_charges_monthly] || false
     end
 
+    def create_usage_threshold(plan, params)
+      usage_threshold = plan.usage_thresholds.new(
+        threshold_display_name: params[:threshold_display_name],
+        amount_cents: params[:amount_cents],
+        amount_currency: plan.amount_currency,
```

Comment:
Another alternative is to do like we do in the `Charge` model: 

``` 
monetize :amount_cents, with_currency: ->(threshold) { threshold.plan.amount_currency }
```

But this works as well of course 👍

Reply:
I think this is even better in the end...

 - no need to store `amount_cents` in the usage_tresholds
 - risking potential conflict in having saved different currencies in `plans` and `usage_tresholds`
 
@nudded @vincent-pochet @lovrocolic @jdenquin